extensions/sql-database-projects/src/tasks/sqlDatabaseProjectTaskProvider.ts

```diff
@@ -101,8 +137,18 @@ export class SqlDatabaseProjectTaskProvider implements vscode.TaskProvider {
 			}
 			const sqlProjPaths = await vscode.workspace.findFiles(new vscode.RelativePattern(folderPath, '**/*.sqlproj'));
```

Comment:
Why does this return duplicates in the first place?

Reply:
We're iterating through the opened workspace folders to locate *.sqlproj files. If a child project is opened first and then its parent is opened later, both become separate workspace folders. In this scenario, the parent workspace returns both its own and the child's .sqlproj files, while the child workspace already includes its own. As a result, tasks for the child project are created twice.